app/src/main/java/org/oppia/app/home/HomeFragmentPresenter.kt

```diff
@@ -18,20 +19,26 @@ import org.oppia.app.home.topiclist.TopicListAdapter
 import org.oppia.app.home.topiclist.TopicSummaryClickListener
 import org.oppia.app.home.topiclist.TopicSummaryViewModel
 import org.oppia.app.model.OngoingStoryList
+import org.oppia.app.model.Profile
+import org.oppia.app.model.ProfileId
 import org.oppia.app.model.TopicList
 import org.oppia.app.model.TopicSummary
 import org.oppia.app.model.UserAppHistory
 import org.oppia.domain.UserAppHistoryController
+import org.oppia.domain.profile.ProfileManagementController
 import org.oppia.domain.topic.TopicListController
 import org.oppia.util.data.AsyncResult
 import org.oppia.util.logging.Logger
+import java.util.*
 import javax.inject.Inject
+import kotlin.collections.ArrayList
 
 /** The presenter for [HomeFragment]. */
 @FragmentScope
 class HomeFragmentPresenter @Inject constructor(
   private val activity: AppCompatActivity,
   private val fragment: Fragment,
+  private val profileManagementController: ProfileManagementController,
   private val userAppHistoryController: UserAppHistoryController,
```

Comment:
Remove the usage of userAppHistory and its controller

Reply:
Its been used couple of times in the same class, not sure whether it is required in future. I will raise an issue related to this to remove unwanted userAppHistoryController #600 .

Reply:
Sounds good. Thanks.